cli/azd/extensions/azure.ai.agents/internal/cmd/init.go

```diff
@@ -144,6 +145,9 @@ func newInitCommand() *cobra.Command {
 	cmd.Flags().StringVarP(&flags.src, "src", "s", "",
 		"[Optional] Directory to download the agent yaml to (defaults to 'src/<agent-id>')")
 
+	cmd.Flags().StringVarP(&flags.host, "host", "", "",
```

Comment:
Do you want to include the default value here? or will you apply it later when you find this is empty?